library/enqueue-scripts.php

```diff
@@ -15,11 +15,14 @@ function foundationpress_scripts() {
 	// Enqueue the main Stylesheet.
 	wp_enqueue_style( 'main-stylesheet', get_template_directory_uri() . '/dist/assets/css/app.css', array(), '2.10.2', 'all' );
 
-	// Deregister the jquery version bundled with WordPress, as we're injecting the version we need with Webpack.
+	// Deregister the jquery version bundled with WordPress.
 	wp_deregister_script( 'jquery' );
 
+	// CDN hosted jQuery placed in the header, as some plugins require that jQuery is loaded in the header.
+	wp_enqueue_script( 'jquery', 'https://ajax.googleapis.com/ajax/libs/jquery/3.2.1/jquery.min.js', array(), '2.10.2', false );
```

Comment:
I think versioning for this should reflect the jQuery version and not FoundationPress. 

So this should be
``wp_enqueue_script( 'jquery', 'https://ajax.googleapis.com/ajax/libs/jquery/3.2.1/jquery.min.js', array(), '3.2.1', false );``